engine/gamesys/src/gamesys/resources/res_sound_data.cpp

```diff
@@ -105,69 +100,30 @@ namespace dmGameSystem
         dmSound::SetSoundDataCallback(resource->m_SoundData, 0, 0);
         dmSound::Result r = dmSound::DeleteSoundData(resource->m_SoundData);
 
+        if (resource->m_Context->m_Cache)
+            ResourceChunkCacheEvictPathHash(resource->m_Context->m_Cache, resource->m_PathHash);
```

Comment:
If it's a streaming resource, then we'll clear out all the chunks associated with it.